system/jlib/jsecrets.cpp

```diff
@@ -350,7 +361,7 @@ class CVault
                 StringBuffer tokenText;
                 if (getSecretKeyValue(clientToken, clientSecret, "token"))
                 {
-                    authType = VaultAuthType::appRole;
+                    authType = VaultAuthType::token;
```

Comment:
Is this a separate bug fix?  Should it be a separate PR, or at least explicitly mentioned in the commit.  (Or did it have no effect?)

Reply:
It was broken, but I doubt this is currently used by anyone but me, although it could be useful at some point.  I'll create a separate JIRA/PR.